unyt/__init__.py

```diff
@@ -96,7 +96,9 @@
 from unyt.unit_systems import UnitSystem  # NOQA: F401
 from unyt.testing import assert_allclose_units  # NOQA: F401
 from unyt.dimensions import accepts, returns  # NOQA: F401
-from unyt import mpl_interface  # NOQA: F401
+from unyt.mpl_interface import matplotlib_support  # NOQA: F401
```

Comment:
This needs either a try/except here or you need to define a dummy `matplotlib_support` in `mpl_interface` in the try/except there. That will fix the test failures on the jobs where we don’t install matplotlib.

It occurs to me that some docs on how to use pyenv and tox to run the full test suite locally would be helpful :)

Reply:
Yes!!!!